
running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s


running 1 test
test check_cached ... FAILED

failures:

---- check_cached stdout ----
Executing expensive function...
thread 'check_cached' panicked at cache_serde/tests/tests.rs:19:5:
assertion `left == right` failed
  left: "Hello2"
 right: "Hello"
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    check_cached

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 2.00s

